core/src/test/resources/sql/hep.iq

```diff
@@ -0,0 +1,108 @@
+# hep.iq - hep tests can customizable optimization rules for hep planner
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This documents the required format for .iq files containing hep planner rule configurations
+#
+# CORRECT SYNTAX:
+# ---------------
+# !set hep-rules "
+# +CoreRules.RULE_NAME1,
+# -CoreRules.RULE_NAME2,
+# +EnumerableRules.ENUMERABLE_RULE_NAME"
+#
+# FORMAT RULES:
+# -------------
+# 1. Must begin with: !set hep-rules " (double quote)
+# 2. Each rule must be on its own line. All lines except last must end with comma.
+#    Final rule line must close with " (double quote) on final line
+# 3. Rules must start with - (remove) or + (add)
+# 4. Rule prefix can be:
+#    - CoreRules.RULE_NAME -> added/removed from hep program
+#    - EnumerableRules.RULE_NAME -> added/removed from volcano program
```

Comment:
so there is both a volcano and a hep program?
you have to describe the order in which the two optimizers are executed.